shared/desktop/renderer/index.js

```diff
@@ -65,6 +63,10 @@ function setupApp(store) {
 
   setupContextMenu(electron.remote.getCurrentWindow())
 
+  // Tell the main window some remote window needs its props
```

Comment:
This comment is wrong? this listens for this event

Reply:
it listens for the event and sends it to the store, which then tells the main window